proxy/src/outbound.rs

```diff
@@ -146,7 +158,7 @@ pub enum Discovery<B> {
 
 impl<B> Discover for Discovery<B>
 where
-    B: tower_h2::Body + 'static,
+    B: tower_h2::Body + 'static
```

Comment:
Just a tiny nit, don't worry about it, but as to *why* there's a trailing comma: adding more bounds would only need to add new lines, not modify this one too :D

Reply:
I didn't mean to remove the trailing comma, whoops!